### ISSUE TITLE
font-family override for buttons and inputs causing padding/margin issues

### ISSUE THREAD

**Issue by haynesjm42** [xh/hoist-react#1489]
Noticed that a Select with multi enabled in a toolbar was always getting a scrollbar, ended up tracking the issue down to font-family overrides that we added on `input` and `button` elements. Also noticed that there are other more subtle padding/margin issues, most noticeable by text on buttons not quite being vertically centered

Haven't tried on a mac, but this is what I am seeing on my Windows 10 machine when toggling the font-family override on and off:

![font-issue](https://user-images.githubusercontent.com/9619538/71186798-d2a2d600-224b-11ea-9500-3d78f45d4e6f.gif)

**Comment by haynesjm42** [comment]
This is the commit which introduced the override: https://github.com/xh/hoist-react/commit/e2b9fe057b03d4de4c2e92d8c26bbdda91ed3af8

@amcclain do we just need to revisit our padding/margin/sizing styling for inputs and buttons now that they are using the correct font?

**Comment by amcclain** [comment]
Thanks for catching this.  On Mac I did not see any such issues, but there are clearly problems when viewing in Chrome on Windows.

I spent a little while trying to understand how some of these styles were breaking. I'm still not clear on it - the way the browsers handle the default user-agent stylesheets and what assumptions appear to be baked into those is a bit of a mystery.  I am simply going to unwind this commit for now. I also found another issue where the button font-size was overriding a more specific button-font-size CSS var we already had applied via a bp- class selector.
